src/Cms/Block.php

```diff
@@ -77,6 +77,12 @@ public function __construct(array $params)
             throw new InvalidArgumentException('The block type is missing');
         }
 
+        // make sure the content is always defined as array to keep
+        // at least a bit of backward compatibility with older fields
+        if (is_array($params['content'] ?? []) === false) {
+            $params['content'] = [];
+        }
+
         $this->content  = $params['content']  ?? [];
```

Comment:
To avoid the duplicated fallback code:

```suggestion
        if (is_array($params['content'] ?? null) === false) {
            $params['content'] = [];
        }

        $this->content  = $params['content'];
```